src/components/layout/Header.vue

```diff
@@ -100,9 +100,12 @@ export default {
         },
         language(){
           this.isLangDropDownOpen = false;
+          this.isHelpDropDownOpen = true;
         },
         help(){
           this.isHelpDropDownOpen = false;
+          this.isLangDropDownOpen = true;
+
```

Comment:
that isnt a solution, also try click the wallet dropdown all 3 dropdowns will open the same time